sdk/keyvault/Azure.Security.KeyVault.Certificates/CHANGELOG.md

```diff
@@ -1,4 +1,11 @@
-# Release History
+# 4.0.0
```

Comment:
Our release notes summarized what was already in 4.0.0-preview.5 below. This is true of all client libraries for Key Vault. If the goal is to clear the current preview release notes, I recommend doing just that (i.e. clear it). We maintain these changes / release notes simultaneously.